src/runtime/hip_helper.cpp

```diff
@@ -238,7 +238,7 @@ __host__ void
   int idx = 0;
   printf("%s", prefix);
   for (idx = 0; idx < num_elements; idx++) {
-    printf(" %.4lf", (float)host_ptr[idx]);
+    printf(" %.20lf", (float)host_ptr[idx]);
```

Comment:
is this just a temporary change, or should we apply this change in the inference branch?

Reply:
just for my my local debugging, revert!